gramps/gen/db/base.py

```diff
@@ -25,21 +25,21 @@
 from this class.
 """
 
-#-------------------------------------------------------------------------
+# -------------------------------------------------------------------------
```

Comment:
I have pylint installed in Windows, version 1.5.5.  My Eclipse development platform also seems to have a pylint like checker built in; it is always complaining about "E265 Block comment should start with a '# ' as well.